pkg/consoleplugin/contentserver/static/plugin-manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "toolchain-member-web-console-plugin",
+  "name": "console-plugin-template",
```

Comment:
I think this is the root cause of the broken changes I mentioned above. We have to keep `toolchain-member-web-console-plugin`.